models2/CommentModel.es6.js

```diff
@@ -27,6 +27,7 @@ export default class CommentModel extends RedditModel {
     replies: T.array,
     numReplies: T.number,
     loadMore: T.bool,
+    loadMoreIds: T.array,
```

Comment:
Whoops, silly me, you can actually do `T.arrayOf` today! https://github.com/reddit/node-api-client/blob/3X/apiBase/Model.es6.js#L41